tests/lib/timespec_utils/testcase.yaml

```diff
@@ -0,0 +1,28 @@
+# FIXME: this should be under tests/unit/timeutil but will not work due to #90029
+common:
+  tags:
+    - timeutils
+  integration_platforms:
+    - native_sim/64
+tests:
```

Comment:
It would also be good to add a testcase to verify the non-builtin code path.

Keeping this in draft until I've done that.